config/locales/en.yml

```diff
@@ -45,9 +45,13 @@ en:
         cities: Popular cities and towns
         counties: English counties
         location_links_heading: Explore teaching jobs by location
+      read_only:
+        title: Sorry, listing a job is unavailable
+        description_html: <p>We’re completing planned maintenance on job listings.</p><p>You can search and view vacancies, but you can’t manage them.</p><p>You’ll be able to use the full service from [insert time/date].</p>
```

Comment:
Don't forget to insert time/date

Reply:
Thanks, I am in talks with content designer to change that to a less time specific information